### DISCUSSION TITLE
Change terraform module to match starship

### DISCUSSION THREAD

**Discussion by ProgRick** [JanDeDobbeleer/oh-my-posh#4510] [General]
The starship prompt contains the following criteria to enable the terraform module:
_By default the module will be shown if **any** of the following conditions are met:

    The current directory contains a .terraform folder
    Current directory contains a file with the .tf, .tfplan or .tfstate extensions_

whereas oh-my-posh only matches if the directory contains a .terraform folder.
Can the logic be modified to also enable terraform if a file with .tf, .tfplan or .tfstate extension is found?

**Comment by JanDeDobbeleer** [comment]
@ProgRick I'll gladly add that. In the meantime, you can add the secret `extensions` property with these values in the terraform segment.

**Reply by JanDeDobbeleer** [reply]
#4512

**Reply by ProgRick** [reply]
Thanks for the quick turn around. Is there a delay before this is available from brew? I just updated and it installed version 18.27.1.

**Reply by JanDeDobbeleer** [reply]
The official brew (like in the docs) is available immediately
